CASMPET-7622: Document iSCSI SBPS configuration in CSM install and Upgrade docs (#6117)
Cray-HPE/docs-csm: operations/iuf/workflows/management_rollout.md

- Host firmware has been updated on management nodes

> Note: Before proceeding with `management-nodes-rollout` iSCSI SBPS configuration has to be done where worker nodes need

Reviewer: This doesn't sound right, is it? They don't need to configure the worker nodes as iSCSI targets before that, do they? I thought they just needed to create the group, so it would know which ones to configure as targets. Or did I misunderstand?

Author: I meant this is the basic step for iSCSI targets configuration.

Reviewer: In that case, I think it should be reworded, for clarity.

to be configured as iSCSI targets for iSCSI SBPS. Unlike in CSM 1.6.0, where all worker nodes were configured as iSCSI

> targets by default without requiring any manual steps, in CSM 1.7.0, selective worker node personalization is introduced

Reviewer: This section also needs to mention that if they DO choose to create the group and limit the workers being configured, then they will have manual DNS steps to perform, right? Making the group only takes care of part of it, I thought.

> to be configured as iSCSI targets for iSCSI SBPS. Unlike in CSM 1.6.0, where all worker nodes were configured as iSCSI
> targets by default without requiring any manual steps, in CSM 1.7.0, selective worker node personalization is introduced

Reviewer: No need to mention the CSM 1.6 patch version in this context. It does make sense to do it for CSM 1.7, so they know exactly when this feature was introduced.

```suggestion
to be configured as iSCSI targets for iSCSI SBPS. Unlike in CSM 1.6, where all worker nodes were configured as iSCSI
targets by default without requiring any manual steps, in CSM 1.7.0, selective worker node personalization is introduced
```

> where it requires to create an HSM group named `iscsi_worker` and add the xnames of worker nodes which are intended to

Reviewer: It's not required to create the group, right? In fact, don't they only need to create it if they don't want every worker to be configured for iSCSI?

Author: This documentation is based on current code (where 'iscsi_worker' HSM group needs to be created) not based on your latest code changes.

> be configured as iSCSI targets. If this is not done, none of the worker nodes will be configured as iSCSI targets. So

Reviewer: Like above, this isn't true, right? If the group isn't created, it means every worker will be configured for iSCSI, unless something changed since we talked about this last.

Author: As I replied to earlier comment, this documentation is about current code changes not based on your latest code changes.

> please refer the steps for the same under `Steps to create HSM group` of [HSM groups iSCSI](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/HSM_groups_iscsi.md).

Reviewer: These links should be relative links within the repo, not https links to github,

> 

Reviewer: This also needs to mention the fact that if they do not choose to limit this now (and therefore all of the workers are configured for iSCSI), it will be more of a pain for them to limit it later, since it will require the manual DNS steps. In other words, some words should be put here to explain why it's worth pausing to decide if they want to do this, rather than figuring they'll take care of it after the upgrade.

> For the details on iSCSI SBPS Please refer [iSCSI SBPS](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/iscsi_sbps.md)

Reviewer 2: With the merge of https://github.com/Cray-HPE/csm-config/pull/381 this will need to change.  Setting up the group is now optional, and only needed to limit iscsi to a subset of the worker nodes.

Author: The current change of ISCSI which will be delivered in RC1 the https://github.com/Cray-HPE/docs-csm/pull/6117/files content is appropriate and later we will have implementation of https://github.com/Cray-HPE/csm-config/tree/CASMPET-7616 delivered and it's associated https://github.com/Cray-HPE/csm-config/pull/381 doc content

> For the details on iSCSI SBPS Please refer [iSCSI SBPS](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/iscsi_sbps.md)

Reviewer: ```suggestion
For more information on iSCSI SBPS, see [iSCSI SBPS](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/iscsi_sbps.md).
```

> Note: Before proceeding with `management-nodes-rollout` iSCSI SBPS configuration has to be done where worker nodes need
> to be configured as iSCSI targets for iSCSI SBPS. Unlike in CSM 1.6.0, where all worker nodes were configured as iSCSI
> targets by default without requiring any manual steps, in CSM 1.7.0, selective worker node personalization is introduced
> where it requires to create an HSM group named `iscsi_worker` and add the xnames of worker nodes which are intended to
> be configured as iSCSI targets. If this is not done, none of the worker nodes will be configured as iSCSI targets. So
> please refer the steps for the same under `Steps to create HSM group` of [HSM groups iSCSI](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/HSM_groups_iscsi.md).
> 
> For the details on iSCSI SBPS Please refer [iSCSI SBPS](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/iscsi_sbps.md)

Reviewer 3: ```suggestion
Note: Before proceeding with the `management-nodes-rollout` stage, the iSCSI SBPS configuration must be performed to designate specific worker nodes as iSCSI targets for iSCSI SBPS. In CSM 1.6.0, all worker nodes were configured as iSCSI
targets by default without requiring any manual steps. Starting in CSM 1.7.0, selective worker node personalization is supported. To enable selective worker node personalization, an HSM group named iscsi_worker must be created, and the xnames of the worker nodes intended to serve as iSCSI targets must be added to this group. If this is not done, none of the worker nodes will be configured as iSCSI targets. See the "Steps to create HSM group" section of the [HSM groups iSCSI](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/HSM_groups_iscsi.md) procedure.

For the details on iSCSI SBPS, see [iSCSI SBPS](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/iscsi_sbps.md).
```
Copy edit of the text.
